arangod/RocksDBEngine/RocksDBBuilderIndex.cpp

```diff
@@ -706,7 +322,9 @@ static arangodb::Result fillIndex(
     // if an error occured drop() will be called
     LOG_TOPIC("dfa3b", DEBUG, Logger::ENGINES)
         << "snapshot captured " << numDocsWritten << " " << res.errorMessage();
-  } else {
+  }
+#ifdef USE_ENTERPRISE
+  else {
     std::deque<std::pair<uint64_t, uint64_t>> partitions;
```

Comment:
Same here. The partitioning code should be in the enterprise repo, because it will never run in the community version.